views/app.js

```diff
@@ -0,0 +1,90 @@
+//var used for global scope availability
+var app = app || {};
+$       = jQuery;
+
+// noinspection JSVoidFunctionReturnValueUsed
+app.MyApp = Backbone.View.extend({
+	el: $('#book-app'),
+	/**
+	 * Initializes MyApp the app with models, views, collections and listeners
+	 */
+	initialize: function () {
+		this.Library   = app.Library;
+		this.BookModel = app.BookModel;
+		this.BookItem  = app.BookItem;
```

Comment:
you don't need to get these props in `MyApp`. You can just use them as `const itemView = new app.BookItem();`

Reply:
I missunderstood smth, I'll try to change that